c++/src/kj/compat/http.c++

```diff
@@ -2359,9 +2374,47 @@ public:
           // Shouldn't get here; handled above.
           KJ_UNREACHABLE;
         case OPCODE_TEXT:
+          KJ_IF_MAYBE(config, compressionConfig) {
+            KJ_DBG("[RECV TEXT] Compressed size", message.size());
+            auto& decompressor = KJ_ASSERT_NONNULL(decompressionContext);
+            auto tail = message.slice(message.size() - 4, message.size());
```

Comment:
Might be worth a short comment here that the extra 4 bytes were added above in the `if (recvHeader.isCompressed()) {` check.

Reply:
https://github.com/capnproto/capnproto/pull/1505/commits/96a1e0d46eeef2f33abfa788011e5707adbfc361